src/main/java/org/cryptomator/ui/fxapp/FxApplication.java

```diff
@@ -68,7 +68,7 @@ public void start() {
 		});
 
 		launchEventHandler.startHandlingLaunchEvents();
-		autoUnlocker.unlock();
+		autoUnlocker.unlockAll();
+		autoUnlocker.startMissingVaultsChecker();
```

Comment:
The missing vault check is an internal feature of the auto unlocker. Hence, there should be only method accessible from the outside which starts autounlock and polling for missing vaults. Also add a parameter, which defines the duration of the unlock tries.

The call would look something like:
```java
   ...
   launchEventHandler.startHandlingLaunchEvents();
   autoUnlocker.tryUnlockForTimespan(2);
}
```